### PR TITLE
v0.20.0

### PR DESCRIPTION
**Pull request by alice-sevin**

### CODE REVIEW

**Comment by alice-sevin** [comment]
[(2) New release du VQB](https://trello.com/c/M2bFvTl4/661-2-new-release-du-vqb)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## Changes
+## [0.20.0] - 2020-08-03
 
 ### Fixed
+- MongoQuery: generate a join step with column names containing any character
```
Comment (davinov) [inline CHANGELOG.md:6]:
```suggestion
- Mongo translator: generate a join step with column names containing any character
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## Changes
+## [0.20.0] - 2020-08-03
```
Comment (davinov) [inline CHANGELOG.md:3]: Could you also add the link to the changelog from this version to the previous one at the end of this file please ?

Comment (alice-sevin) [reply CHANGELOG.md:3]: Of course, my bad!

File: CHANGELOG.md [hunk 3]
```diff
@@ -346,6 +347,7 @@
 
 - Initial version, showtime!
 
+[0.20.0]: https://github.com/ToucanToco/weaverbird/compare/v0.19.2...v0.20.0
```
Comment (davinov) [inline CHANGELOG.md:350]:
```suggestion
[0.20.0]: https://github.com/ToucanToco/weaverbird/compare/v0.19.3...v0.20.0
```

Comment (alice-sevin) [reply CHANGELOG.md:350]: Ah mais je suis à l'ouest ou quoi --', merci <3

**Pull request merged**